scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerHandshaker.java

```diff
@@ -689,7 +689,7 @@ private void createCertificateRequest(final ClientHello clientHello, final DTLSF
 		if (psk == null) {
 			throw new HandshakeException(
```

Comment:
It may be academic, but wasn't the discussion, that in this case, a random secret is assumed to "fake" the calculation times?

Reply:
As we don't answer at all, there is no calculation time to detect.

But if we had chosen to send "decrypt_error" alert, we should have to do something like this. (I mean fake calculation)

This is one of the reason which make this solution simple to implement.

Reply:
:-). If we do a parallel handshake, we may see the difference there :-).